test/example_configs_test_setup.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+set -e
+
+tar -xvf configs/example_configs.tar
```

Comment:
@htuch I'm a little confused by this. AFAICT, I'm decompressing into runfiles, which is read only, right? It seems to work. Should it be failing? I think tomorrow either way I will decompress this into TEST_TMPDIR and expect to load JSON files from there. Is that right?